css/README.md

```diff
@@ -0,0 +1,45 @@
+# CSS Style Guide
+
+## General Guidelines for CSS Code
+
+### Readability
+
+- As with any code, ensure CSS code is optimized for human reading
+- Use descriptive class/variable names and consistent naming conventions
+- Use tools like *prettier* to maintain well-formatted CSS files
+- Leave comments in CSS code when necessary
+
+### Component Libraries
+
+- The usage of component libraries like *Ant Design* or *MUI* is encouraged
+- Integrate with the library's theme system to provide base design and components for your app
+
+### DRY
+
+- Do not copy-paste similar CSS or components
+- Extract common code into a new component where necessary
+- Use utility CSS classes where appropriate
+- Extract variables for reusing common colors, fonts, metrics, etc.
+
+### CSS Modularity
+
+- Avoid using global styles and selectors
+- Keep component's CSS by the component, e.g. using CSS modules
+
+### Pre-processors, CSS in JS
+
+- Feel free to use pre-processors or CSS in JS libraries like *less*, *styled-components* while maintaining best practices
+
+### Responsive Design
```

Comment:
Does it make sense to use 'em' / 'rem' instead of 'px' to support correct zoom behaviour?

Reply:
Good point, however zoom behavior is not affected by using `px`s. Where it might be an accessibility issue is if the user changes the default text size setting in their browser. Thinking in `px`s is easier for developers, I believe we don't use relative units on our projects by default, but it does not mean we shouldn't. Are there any other ideas/experience/preference why to start using relative units?

Reply:
I haven't done it myself, but I agree that avoiding `px` might be good idea - preferring relative units like `(r)em` or percentiles (or `vw` etc...)

Reply:
added to Accessibility section